Commands_Functions/Create_Command.py

```diff
@@ -0,0 +1,33 @@
+import os, json
+from Schema_Keys import Keys
+from Icommand import Icommand
+from output.exceptions import MissingInput, WrongInput
+from output.status import Status
+
+class CreateCommand(Icommand):
+    def __init__(self, schema_path):
+        self.schema_path = schema_path
+    def isvalid(self):
+        if self.schema_path == None or not os.path.exists(self.schema_path) :
+            raise MissingInput(status = Status.MissingInput, message = "the schema is missing")
+        self.schema_file = self.schema_path
+        schema = open(self.schema_path, "r")
+        try:
+            self.schema_data = json.load(schema)
+        except:
+            raise WrongInput(status = Status.WrongInput, message = "schema is written in a wrong json format") 
+        schema.close()
+        return Status.SUCCESS
```

Comment:
It shouldn't return the status. If it doesn't then throw an exception, it should be success. 

if it return status, what will GetCommand().execute return ?